lms/djangoapps/badges/tests/test_models.py

```diff
@@ -33,6 +33,7 @@ def get_image(name):
 
 
 @attr(shard=1)
+@override_settings(MEDIA_ROOT=TEST_DATA_ROOT)
```

Comment:
How was this working before the override?

Reply:
1.10 made some changes to the underlying storage filename handling that I think introduced the check on these paths. The errors this cleaned up were all SuspiciousFileOperations based on the files not being in the root.